src/main/java/se/bjurr/gitchangelog/api/GitChangelogApi.java

```diff
@@ -513,6 +514,33 @@ public GitChangelogApi withRedmineUsername(final String string) {
     return this;
   }
 
+  /**
```

Comment:
I still don't understand this feature. Are you looking for issues that are not mentioned in the commit message?

When would I want to use this method and what would be the result?

Perhaps these are 2 methods are 2 completely separate features? In that case it might be easier to understand the PR if it is split into 2.

Reply:
Great question haha this feature is also not one that I ever need for my team's workflow, but according to the original author of #97:

> This feature is needed, if you develop for e.g. multiple clients in one branch and in one Jira project, but each client should only see the commits that belong to him. So you can define a custom field and integrate this into the plugin configuration and filter out all other tickets that are of no interest.

That made kinda sense to me, maybe?  I believe you would use `search` vs `issue` for when you are wanting an issue returned to be skipped. I tested this w/ Postman by adding essentially an `assignee = 'me@email.com'` and seeing the ticket either showing up or not. 

I am with you that this can be separated into two different PRs, essentially a 'fixed' #97 and then the "additional fields" work I added on top. Since my stuff is what's important to me, this PR will be rebased to only include the additional fields.